charts/bulk-scan-processor/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: bulk-scan-processor
 apiVersion: v1
 home: https://github.com/hmcts/bulk-scan-processor
-version: 0.2.20
+version: 0.2.21
```

Comment:
not a crucial/breaking change. we can bundle up the chart change version bump. unless pipeline actually breaks (afaik the plan was to break)